CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.0
+
+- Thanks to the work of [Bryan Elee](https://github.com/rxbryan) (as part of his Google Summer of Code work) and [Dan Čermák](https://github.com/dcermak), there is a new `packit build in-obs` CLI command to trigger a build in [OBS](https://build.opensuse.org/). Please, try it and let us know how it works and if you find it useful. (#2067)
```

Comment:
I would maybe omit the last sentence for the changelog and maybe rather mention it in the blog/promote it separately.

Reply:
```suggestion
- Thanks to the work of [Bryan Elee](https://github.com/rxbryan) (as part of his Google Summer of Code work) and [Dan Čermák](https://github.com/dcermak), there is a new `packit build in-obs` CLI command to trigger a build in [OBS](https://build.opensuse.org/). (#2067)
```